api/src/services/template.ts

```diff
@@ -44,6 +45,41 @@ class TemplateService {
     return foundTemplateJson;
   }
 
+  async searchTemplates(
+    page: number = 0,
+    range: number = 100,
+    searchMessagesTitleENUS?: string,
+    searchMessagesDescriptionENUS?: string
+  ) {
+    let queryBuilder = Template.query().page(page, range);
+
+    if (searchMessagesTitleENUS) {
+      queryBuilder = queryBuilder.where("messages_title_enUS", "like", `%${searchMessagesTitleENUS}%`)
+    }
+
+    if (searchMessagesDescriptionENUS) {
+      queryBuilder = queryBuilder.where("messages_description_enUS", "like", `%${searchMessagesDescriptionENUS}%`)
```

Comment:
Wondering if it would be possible to do full-text, instead of searching for exact substring matches?

I can see that SQLite does support some kind of full-text search capabilities: https://www.sqlite.org/fts5.html. But I'm not sure if this feature is available in better-sqlite3 which is used in this project.